civihr_employee_portal/features/civihr_employee_portal_features/civihr_employee_portal_features.views_default.inc

```diff
@@ -1968,5 +1968,409 @@ return date(\'Y-m-d\');';
   );
   $export['civihr_report_leave_and_absence'] = $view;
 
+  $view = new view();
+  $view->name = 'calendar_absence_list';
```

Comment:
This view is included in SSP module and is installed with SSP module installation but was not included into Features so I've added it to be able to update its permissions by doing features-revert.